build.sbt

```diff
@@ -707,29 +721,31 @@ val docs = project
     }
   )
 
+val jsBridge06Name = "bloop-js-bridge-0-6"
```

Comment:
The name was actually `bloop-js-bridge-0.6`, but the `.` would get translated to `-`